infrastructure/azure.tf

```diff
@@ -360,30 +386,27 @@ resource "null_resource" "azurerm_cosmosdb_collections" {
 ## APPLICATION INSIGHTS
 
 resource "azurerm_application_insights" "azurerm_application_insights" {
-    name                = "${var.azurerm_application_insights}"
-    location            = "${azurerm_resource_group.azurerm_resource_group.location}"
-    resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
-    # Web or Other
-    application_type    = "Web"
+  name                = "${var.azurerm_application_insights}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+
+  # Web or Other
+  application_type = "Web"
 }
 
 ## APP SERVICE PLAN
 
 resource "azurerm_app_service_plan" "azurerm_app_service_plan" {
-    name                = "${var.azurerm_app_service_plan}"
-    location            = "${azurerm_resource_group.azurerm_resource_group.location}"
-    resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+  name                = "${var.azurerm_app_service_plan}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
 
-    sku {
-        tier = "Standard"
-        # Possible values are B1, B2, B3, D1, F1, FREE, P1, P2, P3, S1, S2, S3, SHARED
-        size = "S1"
-    }
+  sku {
+    tier = "Standard"
 
-    ## !!! FUNCTIONS APP ARE NOT SUPPORTED: we create them manually
-    # provisioner "local-exec" {
-    #    command = "ts-node ./tasks/functions.ts"
-    # }
+    # Possible values are B1, B2, B3, D1, F1, FREE, P1, P2, P3, S1, S2, S3, SHARED
```

Comment:
instead of enumerating the possible values here, it's usually better to link to the relevant doc (as the possible values may change in future versions)